libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -1317,10 +1317,6 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
                 return "themes_details_accessed";
             case ACCOUNT_SETTINGS_LANGUAGE_CHANGED:
                 return "account_settings_language_changed";
-            case SITE_CREATION_BACKGROUND_SERVICE_UPDATE:
```

Comment:
These two events were not being used so I've removed them -> we use `enhanced_site_creation_background_service_updated`.